app/models/concerns/bank_holiday.rb

```diff
@@ -0,0 +1,18 @@
+module BankHoliday
+  include ActiveSupport::Concern
+
+  def is_bank_holiday?
+    read_bank_holidays.include?(self)
+  end
+
+  private
+
+  def read_bank_holidays
+    return [] unless File.exists?(bank_holidays_file)
```

Comment:
Can we write an error to the log if the bank holidays file does not exist please?
Something like ```Rails.logger.error "Bank holidays file missing - rota will not be able to recognise bank holidays"```